exports/@material-ui/core/4.10.1.json

```diff
@@ -113,5 +113,9 @@
   "./Tooltip": "./esm/Tooltip/Tooltip.js",
   "./Typography": "./esm/Typography/Typography.js",
   "./Unstable_TrapFocus": "./esm/Unstable_TrapFocus/Unstable_TrapFocus.js",
-  "./Zoom": "./esm/Zoom/Zoom.js"
+  "./Zoom": "./esm/Zoom/Zoom.js",
```

Comment:
🤔  would this be valid syntax?

```json
  "./*": "./esm/*/*.js"
```

If so this package would be suuuuuper easy to type 😛